src/backends/torch/torchlib.cc

```diff
@@ -375,6 +375,7 @@ namespace dd
   int TorchLib<TInputConnectorStrategy, TOutputConnectorStrategy,
                TMLModel>::train(const APIData &ad, APIData &out)
   {
+    using namespace std::chrono;
```

Comment:
left over ?

Reply:
Ok you just move it, I think `using namespace` should be forbidden :p

Reply:
was there before. Its scope is restricted to only this function, to shorten calls to system_clock etc

Reply:
I think it's better to use `using std::chrono::system_lock`, then, no?